QuantLib/ql/termstructures/yield/piecewisezerospreadedtermstructure.hpp

```diff
@@ -109,7 +109,7 @@ namespace QuantLib {
         registerWith(originalCurve_);
         for (Size i = 0; i < spreads_.size(); i++)
             registerWith(spreads_[i]);
-        if (!originalCurve_.empty())
+        if (!originalCurve_.empty() && dates_.size()>1)
```

Comment:
Instead of bypassing the interpolation (which leaves us with an outdated data member), would it be possible to modify the backward-flat and forward-flat interpolations instead, so that they only require one point instead of two?